mutalyzer/config/__init__.py

```diff
@@ -65,10 +65,9 @@ def _setup(self, from_environment=True):
         self._wrapped = Settings()
         self._wrapped.from_object('mutalyzer.config.default_settings')
         if from_environment:
-            # Todo: We crash if the environment variable is not set. Perhaps
-            #   it is more user-friendly if we fall back on ./settings.py or
-            #   ~/mutalyzer_settings.py or something if it exists.
-            self._wrapped.from_envvar(ENVIRONMENT_VARIABLE)
+            global ENVIRONMENT_VARIABLE
+            if ENVIRONMENT_VARIABLE in os.environ:
+                self._wrapped.from_envvar(ENVIRONMENT_VARIABLE)
```

Comment:
It's probably better to use the `silent` keyword argument for `from_envvar`.

Although I can see one reason for doing the check manually; we can then issue a warning if the environment variable is not set. Probably using `warnings.warn` (see [the docs](https://docs.python.org/2/howto/logging.html#when-to-use-logging)).